db/schema.rb

```diff
@@ -11,27 +11,26 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160629045209) do
+ActiveRecord::Schema.define(version: 20160804083238) do
```

Comment:
Hm, I wonder what we can do to mitigate these whitespace schema changes from coming through all the time.

Is it that we have different editor settings, or just that our schema is old and rails is trying to update it to sane defaults?